sdk/action_parser_funcs.go

```diff
@@ -53,12 +53,41 @@ var (
 		"match":        match,
 		"replace":      replace,
 		"contextValue": contextValue,
+		"default":      dfault,
+		"coalesce":     coalesce,
 	}
 )
 
 type ActionFunc func(ctx context.Context, a *ActionParser, inputs ...interface{}) (interface{}, error)
 
-func replace(_ context.Context, a *ActionParser, inputs ...interface{}) (interface{}, error) {
+func dfault(_ context.Context, a *ActionParser, inputs ...interface{}) (interface{}, error) {
+	switch len(inputs) {
+	case 0:
+		return nil, nil // default
+	case 1:
+		return inputs[0], nil // "var" | default OR default "var"
+	case 2:
+		// second arg is the first input value since in case of | (pipe) it's added as last argument
+		if empty(inputs[1]) {
```

Comment:
usage of the function must be.  ${{ default(input0, input1) }} there is no pipe.
If input0 is not empty, you should return input0

Reply:
Done 🙂